net/http/headers.cpp

```diff
@@ -146,7 +146,7 @@ HeadersBase::KV* HeadersBase::kv_add_sort(KV kv) {
     if ((char*)(begin - 1) <= m_buf + m_buf_size)
         LOG_ERROR_RETURN(ENOBUFS, nullptr, "no buffer");
     auto it = std::lower_bound(begin, kv_end(), kv, HA(this));
-    memmove(begin - 1, begin, sizeof(KV) * (it - begin));
+    // memmove(begin - 1, begin, sizeof(KV) * (it - begin));
     m_kv_size++;
```

Comment:
The compiler deems that it's unsafe to overwrite std:pair's memory via pointer. Because of vtable ?

How to rewrite this line to remove warning? @lihuiba